code/game/machinery/adv_med.dm

```diff
@@ -371,6 +372,8 @@
 	dat += text("[]\t-Burn Severity %: []</font><br><br>", (occ["fireloss"] < 60 ? "<font color='blue'>" : "<font color='red'>"), occ["fireloss"])
 
 	dat += text("[]\tRadiation Level %: []</font><br>", (occ["rads"] < 10 ?"<font color='blue'>" : "<font color='red'>"), occ["rads"])
+	if(occ["radtick"] > 0)
+		dat += text("<font color='red'>Absorbed rads danger index: <b>[occ["radtick"]]</b></font><br>")
```

Comment:
i know rads is rad but why not 'radiation'

Reply:
Guess I was trying to shorten the name. Radiation is fine though.

Reply:
Although, you know, a rad is an actual measurement of absorbed radiation dose. Wasn't really thinking of that at the time but it's somewhat appropriate.